pkg/backup/remap_crd_version_action_test.go

```diff
@@ -58,4 +59,23 @@ func TestRemapCRDVersionAction(t *testing.T) {
 		require.NoError(t, err)
 		assert.Equal(t, "apiextensions.k8s.io/v1beta1", item.UnstructuredContent()["apiVersion"])
 	})
+
+	t.Run("Having an integer on a float64 field should work (issue 2319)", func(t *testing.T) {
+		b := builder.ForV1CustomResourceDefinition("test.velero.io")
+		schema := builder.ForJSONSchemaPropsBuilder().Maximum(5).Result()
```

Comment:
magic number `5`

Reply:
I think this is the test data (i.e. an integer in the float field).

Reply:
Correct, this is test data for an int in a float field.

Reply:
Added comment here to explain it